pom.xml

```diff
@@ -769,7 +769,67 @@
 
     </plugins>
     <pluginManagement>
-      <plugins>
+	      <plugins>
```

Comment:
@erdemedeiros I did import the Eclipse IDE configuration, but it is only for Java editor. I need XML Editor configuration for Eclispe to match XML formatting standard in Activiti.